apps/prairielearn/src/middlewares/AuthzAccessMismatch.tsx

```diff
@@ -17,65 +35,65 @@ interface PermissionData extends PermissionMeta {
   authnValue: boolean | string;
 }
 
-const PermissionsMeta = [
+export const PERMISSIONS_META = [
   {
     label: 'Administrator',
     key: 'is_administrator',
     type: 'boolean',
   },
   {
-    label: 'Course preview',
+    label: 'Course role',
+    key: 'course_role',
+    type: 'string',
+  },
+  {
+    label: 'Course previewer',
     key: 'has_course_permission_preview',
     type: 'boolean',
   },
   {
-    label: 'Course view',
+    label: 'Course viewer',
     key: 'has_course_permission_view',
     type: 'boolean',
   },
   {
-    label: 'Course edit',
+    label: 'Course editor',
     key: 'has_course_permission_edit',
     type: 'boolean',
   },
   {
-    label: 'Course own',
+    label: 'Course owner',
     key: 'has_course_permission_own',
     type: 'boolean',
   },
   {
-    label: 'Student access',
-    key: 'has_student_access',
-    type: 'boolean',
-  },
-  {
-    label: 'Enrollment student access',
-    key: 'has_student_access_with_enrollment',
-    type: 'boolean',
+    label: 'Course instance role',
+    key: 'course_instance_role',
+    type: 'string',
   },
   {
-    label: 'Course instance view',
+    label: 'Student data viewer',
```

Comment:
While "course instance ..." is technically correct, we always talk about this as "student data ... permissions".